wgsl/index.bs

```diff
@@ -8185,14 +8185,75 @@ Note: The [=attribute/const=] attribute cannot be applied to user-declared funct
     * An [[#address-of-expr|address-of expression]] of a
         [[#var-identifier-expr|variable identifier expression]]
     * A [=formal parameter|function parameter=]
+    * A [[#constant-identifier-expr|constant identifier expression]] or
```

Comment:
The direction is good, but the 'that is' is not an equivalence.

E.g. you can chain & and * together

```
fn (p : ptr<function,i32>) {
     helper(  &*p );
}
```

We already have the concept of 'full pointer' that is almost exactly what we need.  Except we'd need to fold in the formal parameter case, which then also has to mix into the recursive cases.
It's like we would need concepts of 'full reference to a root identifier' and 'full pointer to a root identifier' that are recusrively defined just like full-reference and full-pointer.

Reply:
That type of example is included the added examples.

Why do you think it is not equivalent? The way we define a memory view is memory locations, type, and access mode. So `p` and `&*p` should be the same view no?